src/prebid.js

```diff
@@ -473,9 +473,9 @@ $$PREBID_GLOBAL$$.onEvent = function (event, handler, id) {
 };
 
 /**
- * @param {String} event the name of the event
+ * @param {string} event the name of the event
```

Comment:
so confused with the lowercase string and uppercase Object

Reply:
I know!  The change is based on the output of `documentation lint`, and also in the [Google Closure Compiler JSDoc guidelines](https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler) FWIW.

Reply:
that's why linters exist in the first place :).

Sometimes it's for consistency. Sometimes it's because your code is subtly incorrect.

My guess is that they were expecting to support Classes in the newer versions of JS, and the general principle was "primitives are lowercase. Classes are uppercase." Just speculation though.